src/main/kotlin/org/rust/ide/surround/RustSurroundDescriptor.kt

```diff
@@ -0,0 +1,29 @@
+package org.rust.ide.surround
+
+import com.intellij.lang.surroundWith.SurroundDescriptor
+import com.intellij.lang.surroundWith.Surrounder
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import com.intellij.psi.PsiWhiteSpace
+import com.intellij.psi.util.PsiTreeUtil
+import org.rust.lang.core.psi.RustExpr
+
+class RustSurroundDescriptor : SurroundDescriptor {
+
+    companion object {
+        private val surrounders = arrayOf(
+            DelimiterSurrounder("(", ")", "Surround with ()"),
+            DelimiterSurrounder("{", "}", "Surround with {}")
+        )
+    }
+
+    override fun isExclusive() = false
+
+    override fun getElementsToSurround(file: PsiFile?, startOffset: Int, endOffset: Int): Array<out PsiElement> {
```

Comment:
I think there is a missing `NotNull` annotation in the Java super method. So it's better to change `file: PsiFile?` to `file: PsiFile` and change `file as PsiFile` to simply `file`.